auditlog/models/rule.py

```diff
@@ -534,11 +586,14 @@ def _get_field(self, model, field_name):
                 cache[model.model][field_name] = field_data
         return cache[model.model][field_name]
 
-    def _create_log_line_on_read(self, log, fields_list, read_values):
+    def _create_log_line_on_read(
+        self, log, fields_list, read_values, fields_to_exclude
+    ):
         """Log field filled on a 'read' operation."""
         log_line_model = self.env["auditlog.log.line"]
+        fields_to_exclude.extend(FIELDS_BLACKLIST)
```

Comment:
```suggestion
        _fields_to_exclude = fields_to_exclude + FIELDS_BLACKLIST
```
Redefine the var inside the method, otherwise you are modifying the argument you received and it might make debugging tricky.